procedures/foreman/release.md.erb

```diff
@@ -69,17 +69,29 @@ Note: If for some reason there was an issue with the tarballs that required uplo
   - [ ] [rpm/<%= short_version%>](https://github.com/theforeman/foreman-packaging/pulls?q=is%3Apr+is%3Aopen+base%3Arpm%2F<%= short_version %>)
   - [ ] [deb/<%= short_version%>](https://github.com/theforeman/foreman-packaging/pulls?q=is%3Apr+is%3Aopen+base%3Adeb%2F<%= short_version %>)
 - Sign the RPMs in the release
+<% if Gem::Version.new(foreman_version) < Gem::Version.new('3.9') -%>
   - [ ] <%= rel_eng_script('download_rpms') %>
   - [ ] <%= rel_eng_script('sign_rpms') %>
   - [ ] <%= rel_eng_script('upload_rpm_signatures') %>
   - [ ] <%= rel_eng_script('upload_rpms') %>
+<% else -%>
+  - [ ] <%= rel_eng_script('generate_stage_repository') %>
+  - [ ] <%= rel_eng_script('sign_stage_rpms') %>
+  - [ ] <%= rel_eng_script('upload_stage_rpms') %>
+<% end -%>
 - Sign RPMs for client repos (call scripts with `PROJECT=client`)
+<% if Gem::Version.new(foreman_version) < Gem::Version.new('3.9') -%>
   - [ ] <%= rel_eng_script('download_rpms') %>
   - [ ] <%= rel_eng_script('sign_rpms') %>
   - [ ] <%= rel_eng_script('upload_rpm_signatures') %>
   - [ ] <%= rel_eng_script('upload_rpms') %>
+<% else -%>
+  - [ ] <%= rel_eng_script('generate_stage_repository') %>
+  - [ ] <%= rel_eng_script('sign_stage_rpms') %>
+  - [ ] <%= rel_eng_script('upload_stage_rpms') %>
+<% end -%>
 - [ ] Kick off the [release pipeline](https://ci.theforeman.org/job/foreman-<%= short_version %>-release-pipeline/) by calling <%= rel_eng_script('release_pipeline') %>
-- [ ] Kick off the [client pipeline](https://ci.theforeman.org/job/foreman-client-<%= short_version %>-rpm-pipeline/) by calling `PROJECT=client ./release_pipeline`
+- [ ] Kick off the [client pipeline](https://ci.theforeman.org/job/foreman-client-<%= short_version %>-rpm-pipeline/) by calling `PROJECT=client <%= rel_eng_script('release_pipeline') %>`
```

Comment:
I'd expect no diff on this line:
```suggestion
- [ ] Kick off the [client pipeline](https://ci.theforeman.org/job/foreman-client-<%= short_version %>-rpm-pipeline/) by calling `PROJECT=client ./release_pipeline`
```

Reply:
I wonder if `rel_eng_script_link` would have been a better name. Because it renders a Markdown link, but that doesn't work inside a code block.